pyproject.toml

```diff
@@ -17,8 +17,32 @@ classifiers = [
     "Operating System :: POSIX",
 ]
 requires-python = ">=3.11"
-# Use requirements/main.in for runtime dependencies instead.
-dependencies = []
+dependencies = [
+    "idna>=3.7",
+    "setuptools>70.0",
+    "certifi>2024.2.0",
+    "pillow>=10.3.0",
+    "ecdsa>=0.18",
+    "BeautifulSoup4",
+    "click",
+    "jinja2>=3.1.4",
+    "marshmallow",
+    "pandoc",
+    "pypandoc",
+    "requests",
+    "setuptools_scm",
```

Comment:
Surely docsteady doesn't depend on setuptools_scm, tox, pre-commit and mypy?

Reply:
it carried a lot of depdancies - i may be able to trim ..